osu.Game/OsuGameBase.cs

```diff
@@ -85,7 +83,7 @@ public class OsuGameBase : Framework.Game, ICanAcceptFiles
         [Cached(Type = typeof(IBindable<IReadOnlyList<Mod>>))]
         protected readonly Bindable<IReadOnlyList<Mod>> Mods = new Bindable<IReadOnlyList<Mod>>(Array.Empty<Mod>());
 
-        protected Bindable<WorkingBeatmap> Beatmap => beatmap;
+        protected Bindable<WorkingBeatmap> Beatmap { get; private set; } // cached via load() method
```

Comment:
Not sure about this. Since there's no attribute, I just copied the comment.

Reply:
I don't really mind it.